### ISSUE TITLE
md-iop.a.3: Encoding should allow gmx:Anchor too

### ISSUE THREAD

**Issue by heidivanparys** [inspire-eu-validation/ets-repository#127]
md-iop.a.3: Encoding is too narrow, the test should allow the presence of gxm:Anchor, which can be used instead of gco:CharacterString in order to allow a link (e.g. to where the specification can be downloaded) to be present.

[The abstract test](https://github.com/inspire-eu-validation/data/blob/3.0rc3/interoperability-metadata/encoding.md) should also be changed, as gco:CharacterString is mentioned there too.

E.g.
```
        <gmd:MD_Format>
          <gmd:name>
            <gco:CharacterString>My name</gco:CharacterString>
          </gmd:name>
          <gmd:version>
            <gco:CharacterString>My version</gco:CharacterString>
          </gmd:version>
          <gmd:specification>
            <gmx:Anchor xlink:href="http://link.to.my.specification.eu">My specification</gmx:Anchor>
          </gmd:specification>
        </gmd:MD_Format>
```

- Test Report: http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID00cc85b0-08bf-4063-b1a4-6a2bb69b994a.html
- Assertion URI: http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID00cc85b0-08bf-4063-b1a4-6a2bb69b994a.html?lang=en#EIDff8b0a0c-2c22-4a4a-bc2f-bbd3257777cc
- [encoding.txt](https://github.com/inspire-eu-validation/ets-repository/files/1501923/encoding.txt)

**Comment by cportele** [comment]
I agree, it should be changed - first in the ATS and then in the ETS.

Probably a general review of the Metadata test cases should be done, if other test cases should be updated, too.

Maybe one point of discussion in the 2016.3 subgroup should be whether the XPath expressions should be

`gmd:distributionInfo/gmd:MD_Distribution/gmd:distributionFormat/gmd:MD_Format/gmd:specification/*`

or

`gmd:distributionInfo/*/gmd:distributionFormat/*/gmd:specification/*`

to allow other substitution elements earlier in the path, too.

**Comment by heidivanparys** [comment]
@michellutz I think this issue should have the "for-2017.4-discussion" label.

**Comment by michellutz** [comment]
I agree. The MD TG v1.3 uses the less strict xPath expressions in the tables for the ISO elements to be used. Therefore they should also be used in the ATS, ETS (incl. for the interoperability metadata elements).

**Comment by michellutz** [comment]
[2017.4 meeting 2016-03-16] The proposed solution to use less strict xPath expressions was agreed. The JRC contractors (Bilbomatica+Guadaltel) will review ATS and ETS and propose changes.

**Comment by michellutz** [comment]
The implementation for the specific case is probably ok. But the agreement was to review the ATS and ETS _in general_ and propose changes to _all_ tests.

**Comment by michellutz** [comment]
@inakidiazdecerio Please see above. The agreement was to review the ATS and ETS in general and propose changes to all tests.
